Automattic/themes
# Update the Calm Business theme to support FSE header

#### Changes proposed in this Pull Request:

This change will update the calm business theme to pull in the header styles into the editor through the style-editor.scss file. This means that when using the full site editing plugin, you will see almost a one to one representation of the header design in the editor vs the front end:

<img width="1295" alt="Screen Shot 2019-07-17 at 12 50 06 PM" src="https://user-images.githubusercontent.com/1464705/61406885-a694ba80-a891-11e9-817c-e8297621df65.png">

To test this:

* Install the calm business theme locally
* Install the latest master branch build of the full site editing plugin from Calypso
* Create a new page
* Edit the header of the page and delete all content
* Paste the following into the editor: https://codepen.io/apeatling/pen/rEbvMN and update/publish the header template.
* Confirm that you see a header that matches the screenshot above, and this looks identical on the front end of the site.
* Confirm that with the full site editing plugin disabled, the old template file based header works as expected.

Any issues with the content width and navigation styles will be addressed separately.

#### Related issue(s):

* https://github.com/Automattic/wp-calypso/issues/34387 
* https://github.com/Automattic/wp-calypso/pull/34568
